services/ui-src/src/utils/featuresByYear.ts

```diff
@@ -100,4 +100,12 @@ export const featuresByYear = {
   get hasStreamlinedOms() {
     return getMeasureYear() >= 2023;
   },
+  /**
+   * Prior to 2025, we display FFY before the year name.
+   *
+   * In 2025 and beyond, we will not display the FFY language in front of the year.
+   */
```

Comment:
I'm going to be really nitpicky about the comment and request some more "why". Something like this:

```suggestion
  /**
   * Prior to 2025, we displayed FFY before the year name. This was done in part
   * to differentiate Federal Fiscal Year and HEDIS Medical Year.
   *
   * In 2025, stakeholders decided this explicit differentiation was unneeded.
   */
```

(and it isn't really _super_ important, but this comment would also bring the max line length back down to 80 chars)